plugins/offboard/include/plugins/offboard/offboard.h

```diff
@@ -146,13 +154,19 @@ class Offboard : public PluginBase {
      */
     void set_velocity_ned(VelocityNEDYaw velocity_ned_yaw);
 
+
```

Comment:
The file generally has one empty line between declarations, I'd keep consistent with that.